chore(grid):sample project with dapper (#129)
telerik/blazor-ui: grid/dapper/README.md

8. Copy the generated connection string and close the window
![eighth step screenshot](images/eighth-step.png)
9. Paste the connection string in the `appsettings.Development.json` file and you are ready to go

## The Magic
If you are looking for where the majority of the code that performs this is held, take a look at `TelerikBlazorGrid_Dapper.DataAccess.Extensions.SqlBuilderExtensions`. We build up 2 sql templates, one for the query to get our data, and another to query the total count.
By default, you should only need to provide this method the table (or view) name you wish to query. However there are instances where you wish to query a different table or view for both the data selection and the count - For example where you have a view containing a join as the count will become inaccurate, counting all the duplicate rows.
You may also specify a different default sort field and direction.

## The Result
This picture shows the end result. A table with full sort, filter and paging functionality - Using OnRead through dapper, and passing the handling of all the Grid functionalities over to the database.
![dapper grid](images/dapper-grid.png)

## Disclaimer

> The Telerik components integration with Dapper is not supported. This is only an example of a custom scenario.

Reviewer: ```suggestion
Integration with Dapper is not subject to official technical support.
```
